Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -1299,7 +1299,7 @@ public AndConstraint<TAssertions> HaveCountGreaterThanOrEqualTo(int expected, st
                 .Then
                 .Given(subject => subject.Count())
                 .ForCondition(actualCount => actualCount >= expected)
-                .FailWith("but found {0}.", actualCount => actualCount)
+                .FailWith("but found {0}: {1}.", actualCount => actualCount, actualCount => Subject)
```

Comment:
🙃  It would probably use the `_` if I did not need the `actualCount`, but that's just me nitpicking